ui/plugin-lifecycle/src/main/interfaces/Tenant.ts

```diff
@@ -0,0 +1,40 @@
+export interface Tenant {
+    canPublishCatalogs: boolean;
+    deployedVMQuota: number;
+    displayName: string;
+    href: string;
+    id: string;
+    isEnabled: boolean;
+    isReadOnly: boolean;
+    link: string[];
+    metadata: string;
+    name: string;
+    numberOfCatalogs: number;
+    numberOfDisks: number;
+    numberOfGroups: number;
+    numberOfVApps: number;
+    numberOfVdcs: number;
+    otherAttributes: {
+        numberOfRunningVMs: string;
+    };
+    storedVMQuota: number;
+    type: string;
+}
+
+export interface UiTenantResponse {
+    href: string;
+    link: string[];
+    name: string;
+    otherAttributes: any;
+    page: number;
+    pageSize: 25;
+    record: Tenant[];
+    total: number;
+    type: string;
+    vcloudExtension: any[];
+}
+
+export interface UiPluginTenantsResponse {
```

Comment:
should this just be an EntityReference2?

Reply:
Changed that in last commit.